gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -124,55 +112,189 @@ object Versions {
-    val roasterJdt             = "org.jboss.forge.roaster:roaster-jdt:${Versions.roaster}"
-    val animalSniffer          = "org.codehaus.mojo:animal-sniffer-annotations:${Versions.animalSniffer}"
-    val ci = "true".equals(System.getenv("CI"))
+    const val runner  = "org.junit.jupiter:junit-jupiter-engine:${version}"
+    const val pioneer = "org.junit-pioneer:junit-pioneer:${pioneerVersion}"
+    const val platformCommons = "org.junit.platform:junit-platform-commons:${platformVersion}"
+}
+
+// https://github.com/JetBrains/kotlin
+// https://github.com/Kotlin
+object Kotlin {
+    @Suppress("MemberVisibilityCanBePrivate") // used directly from outside
+    const val version      = "1.4.21"
+    const val reflect      = "org.jetbrains.kotlin:kotlin-reflect:${version}"
+    const val stdLib       = "org.jetbrains.kotlin:kotlin-stdlib:${version}"
+    const val stdLibCommon = "org.jetbrains.kotlin:kotlin-stdlib-common:${version}"
+    const val stdLibJdk8   = "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${version}"
+}
+
+// https://github.com/protocolbuffers/protobuf
+object Protobuf {
+    @Suppress("MemberVisibilityCanBePrivate") // used directly from outside
+    const val version    = "3.13.0"
+    const val gradlePluginVersion = "0.8.13"
+    val libs = listOf(
+        "com.google.protobuf:protobuf-java:${version}",
+        "com.google.protobuf:protobuf-java-util:${version}"
+    )
+    const val compiler = "com.google.protobuf:protoc:${version}"
+    const val gradlePlugin = "com.google.protobuf:protobuf-gradle-plugin:${gradlePluginVersion}"
+}
+
+// https://github.com/forge/roaster
+object Roaster {
+    private const val version = "2.21.2.Final"
+    const val api     = "org.jboss.forge.roaster:roaster-api:${version}"
+    const val jdt     = "org.jboss.forge.roaster:roaster-jdt:${version}"
+}
+
+// https://github.com/google/truth
+object Truth {
+    private const val version = "1.1.2"
+    val libs = listOf(
+        "com.google.truth:truth:${version}",
+        "com.google.truth.extensions:truth-java8-extension:${version}",
+        "com.google.truth.extensions:truth-proto-extension:${version}"
+    )
+}
+
+/*
+ * Objects below gather dependencies declared above into the groups by purpose.
+ */
+
+object GradlePlugins {
+    const val errorProne  = ErrorProne.gradlePlugin
+    const val protobuf    = Protobuf.gradlePlugin
+    const val appengine   = AppEngine.gradlePlugin
+    val licenseReport = "com.github.jk1:gradle-license-report:${Versions.licensePlugin}"
+}
+
+object Build {
+    val animalSniffer = "org.codehaus.mojo:animal-sniffer-annotations:${Versions.animalSniffer}"
+    const val autoCommon = AutoCommon.lib
+    val autoService = AutoService
+    const val appEngine = AppEngine.sdk
+    val checker = CheckerFramework
+    val errorProne = ErrorProne
+    val firebaseAdmin = "com.google.firebase:firebase-admin:${Versions.firebaseAdmin}"
+    val flogger = Flogger
+    const val guava = Guava.lib
```

Comment:
I would suggest `Deps.guava.lib` instead of `Deps.build.guava`. Or possibly, `Deps.main.guava.lib` or `Deps.prod.guava.lib`, if we want to preserve the grouping of the dependencies into main/test/etc.